client/cli/go/cmds/heketi_storage.go

```diff
@@ -55,6 +57,15 @@ func init() {
 		"image",
 		"heketi/heketi:dev",
 		"container image to run this job")
+	setupHeketiStorageCommand.Flags().StringVar(&heketiStorageDurability,
+		"durability",
+		"replicate",
+		"Durability of the database storage")
+	setupHeketiStorageCommand.Flags().IntVar(&heketiStorageReplicaCount,
+		"replica",
+		3,
+		"\n\tReplica value for durability type 'replicate'."+
+			"\n\tDefault is 3")
```

Comment:
Here the formatting (line break / indentation) is a little inconsistent with the --durability option.
I suggest being a little more verbose and more or less copying the description of
--durability from the "volume create" command (omitting the disperse type).

Reply:
Done!